src/command.ts

```diff
@@ -8,4 +8,5 @@ export interface Command {
 	readonly hasPermission: (state: State, interaction: CommandInteraction) => boolean;
 	readonly shouldBeEphemeral: (state: State, interaction: CommandInteraction) => boolean;
 	readonly handler: (state: State, interaction: CommandInteraction, args: CommandInteractionOption[]) => void;
+	readonly setOauthToken?: (oauthToken: string) => void;
```

Comment:
Instead of adding a new optional field to command that's only applicable to a single command, create a module (e.g., `store/githubOauth.ts`) that can be used to encapsulate the oauthToken which is set in `index.ts` and read from whenever the oauthToken is needed.

Reply:
5b87a1a